backend/backend/settings.py

```diff
@@ -127,3 +128,8 @@
 # https://docs.djangoproject.com/en/3.1/howto/static-files/
 
 STATIC_URL = '/static/'
+
+
+GRAPHENE = {
+    "SCHEMA": "api.schema.schema"
+}
```

Comment:
All other files have a new line at the end. This one doesn't, which is inconsistent.